src/main/java/fr/quatrevieux/araknemu/game/handler/account/SelectCharacter.java

```diff
@@ -66,7 +72,8 @@ public void handle(GameSession session, ChoosePlayingCharacter packet) {
         session.log().ifPresent(log -> log.setPlayerId(player.id()));
 
         session.send(Error.welcome());
-        session.log().flatMap(SessionLog::last).ifPresent(log -> session.send(Information.lastLogin(log.startDate(), log.ipAddress())));
+        System.out.println(configuration.timezone());
```

Comment:
You've forget this :stuck_out_tongue:

Reply:
Fixed in commit [#7f9e93d](https://github.com/Arakne/Araknemu/pull/309/commits/7f9e93dccae08a8befd6b00dc959dc869c517ce7#r1349931391)